<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointCreator</class>
 <widget class="QDialog" name="PointCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="m_cancelBtn">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>171</y>
     <width>80</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>112</y>
     <width>60</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Координата Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>112</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Координата Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>85</y>
     <width>129</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Координаты точки</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>147</y>
     <width>132</width>
     <height>77</height>
    </rect>
   </property>
   <property name="title">
    <string>Вектор нормали</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="m_xNormale">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="m_yNormale">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="m_zNormale_2">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="m_okBtn">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>171</y>
     <width>80</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ок</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_zCoordinate">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>112</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_yCoordinate">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>112</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_xCoordinate">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>112</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>112</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Координата X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>58</y>
     <width>77</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя точки:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pointName">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
